ui/docs/Test.md

```diff
@@ -53,9 +53,21 @@ React component code:
 - `ui/src/Groups`
 - `ui/src/Panels`
 
-End to end test cases validating end user flows should be tested using 
-_<e2e framework here>_. These tests should be defined and sit alongside the
-code in the `ui/src/Bootstrap` directory.
+#### End to End integration test
+
+End to End integration tests should exercise a user scenario - focus on the behaviour a user will take to interact with the application to achieve an end goal.
+
+They should be fairly coarse grained, exercising multiple pages/panels as necessary.
+
+For example - "I can produce a message and observe it being consumed", or "I can stop producing and no more messages are consumed".
+
+These tests are using _Jest_ as a test runner, using standard Jest test assets,
+such as `describe`, `it` and `expect`, and _Playwright_ to instrument the browser.
+
+To avoid the temptation to correlate scenarios with particular panels, these tests are located in the `e2e/test` directory.
```

Comment:
No issue with this - would you mind checking if there is any mention of e2e tests in any of the other .md files/the Bootsrtrap folder readme and updating as required?

Reply:
Sure, on it :) Will address in the cucumber PR if that's ok?

Reply:
Given there is bound to be more churn (and you are the only one on this atm) I dont mind!